.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,37 @@
-*To help us understand & test your amazing contribution, please include the below information with your PR. 🙏 Thanks!*
+<!-- Employees: Please use Clubhouse's "Open PR" button from the relevant story or include links to relevant Clubhouse stories in your branch name, commit messages, or pull request comments. Do not add links to your pull request description, they will be ignored. https://help.clubhouse.io/hc/en-us/articles/207540323-Using-The-Clubhouse-GitHub-Integration -->
 
-
-### Closes #ISSUE_NUMBER_GOES_HERE
+<!-- Employees: Delete this section. -->
+## Closes #ISSUE_NUMBER_GOES_HERE
 
 🎉 🎉 🎉
 
-### Description
+## Description
 
 ......
 
-
-### Changes
+## Changes
 
 * High level
 * changes that
 * you made
 
+### Reusable Code/Components
 
-### Steps to Test
+<!-- Preferably, include automated tests instead. -->
+## Steps to Test
 
 1. Steps
 2. to
 3. test
 
+<!-- Contributors: Delete this section. -->
+## [Zeplin Design](https://app.zeplin.io/project/5b0334f5e91e8c481645ad56)
+<!-- Upload screenshots here. -->
 
-### Screenshots
+## Quality Assurance
 
-...only if applicable...
+- [ ] The branch name is in lowercase-kebab-case with no prefix (unless it was created from Clubhouse)
+- [ ] The base branch is develop or gau (no nested branches)
+- [ ] This is related to a maximum of one Clubhouse story or GitHub issue
+- [ ] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
+- [ ] If code is copied from existing directories, there is an explanation of why this is necesary in the description/changes, and all copying is done in separate commits to make them easy to filter out
```

Comment:
I don't understand the filtering requirement? To simplify code review? Even a copy/paste from an existing codebase should be subject to our new standards.

Reply:
By filtering I mean using the select commits dropdown to see all commits except the ones that copy files:

![image](https://user-images.githubusercontent.com/927220/58660762-2ff92900-82f4-11e9-8d3d-3280036b79b0.png)

The problem is less about code quality, and more that it's difficult to review what changed when a file is copied unless the copy and change are done in separate commits (in which case you can just hide the copy commit in the dropdown to focus on changes).

Reply:
Understood. But I'm still inclined to judge a copy/paste as if it was new code and ask the reviewer to reevaluate it. Ignoring on only looking at changes of copied code would nudge the reviewer to overlook them.

(I'm just trying to stay KISS and keep conventions down to the ones that we can actually automate)

Reply:
That's a valid point, but you can still do that with separate commits, and the separate commits make it easier to review in isolation.

Reply:
I don't see the benefit and doubt this would be respected, especially if we can't enforce it. As you wish